src/sql/base/browser/ui/scrollableSplitview/heightMap.ts

```diff
@@ -89,10 +89,10 @@ export class HeightMap {
 
 	// Contiguous items
 	public onRemoveItems(iterator: INextIterator<string>): void {
-		let itemId: string;
+		let itemId: string | null = null;
```

Comment:
null vs. undefined?

Reply:
it could be undefined, this is based on a vscode component so I just copied theirs. I can probably change it to undefined.

Reply:
For code in VS Code source I think it's better to leave unchanged.  I guess it doesn't matter, we need to always check for both null && undefined anyways.